app/views/groups/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1> All groups </h1>
+  <% if @groups.empty? %>
+    <p>No group added yet!</p>
+  <% end %>
+
+<% @groups.each do |group| %>
+    <p>Name of group: <%= group.name %>
+    <%= button_to "Delete group", user_group_path(current_user, group), method: :delete %>
+    <p><%= link_to "Group detail", user_group_path(current_user, group) %></p>
+<% end %>
+    <p><%= link_to "Back to user", users_path %></p>
```

Comment:
- [ ] Following the project requirements, for each category, the user should see the total amount of all the transactions that belong to that category.